nucypher/blockchain/eth/domains.py

```diff
@@ -63,18 +70,26 @@ def is_testnet(self) -> bool:
     name="mainnet",
     eth_chain=EthChain.MAINNET,
     polygon_chain=PolygonChain.MAINNET,
+    condition_chains=(EthChain.MAINNET, PolygonChain.MAINNET),
 )
 
 LYNX = TACoDomain(
     name="lynx",
     eth_chain=EthChain.GOERLI,
     polygon_chain=PolygonChain.MUMBAI,
+    condition_chains=(
+        EthChain.MAINNET,
+        EthChain.GOERLI,
+        PolygonChain.MUMBAI,
+        PolygonChain.MAINNET,
+    ),
 )
 
 TAPIR = TACoDomain(
     name="tapir",
     eth_chain=EthChain.SEPOLIA,
     polygon_chain=PolygonChain.MUMBAI,
+    condition_chains=(EthChain.SEPOLIA, PolygonChain.MUMBAI),
```

Comment:
Sorry if dumb question, but why LYNX condition chains are 4 and TAPIR condition chains are only SEPOLIA and MUMBAI?

Reply:
I think we should drop mainnet chains only after we deploy TACo.